railties/lib/rails/test_unit/test_requirer.rb

```diff
@@ -15,7 +15,7 @@ def require_files(patterns)
       private
         def expand_patterns(patterns)
           patterns.map do |arg|
-            arg = arg.gsub(/:(\d+)?$/, '')
+            arg = arg.gsub(/:(\d+)?/, '')
```

Comment:
Can we match a series of `:4:9` at the end of a line, instead of checking the whole string?